cypress/support/commands.js

```
@@ -23,3 +23,10 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+Cypress.Commands.add('login', (email, password) => {
```

The `cy.login` command does not validate its arguments. If `email` or `password` is undefined the command will still try to type `undefined`, producing confusing failures. Add an early check and throw a clear error when required arguments are missing (so tests fail fast with a helpful message).